lib_forceload/data/minecraft/loot_tables/blocks/yellow_shulker_box.json

```diff
@@ -2,68 +2,109 @@
   "type": "minecraft:block",
   "pools": [
     {
-      "rolls": 1,
```

Comment:
It's modified in a way that keeps functionality the same, so even if this loot table overwrites the old one, all data packs reliant on it will not show any effect.